help/destinations/catalog/advertising/acxiom-prospect-suppression.md

```diff
@@ -0,0 +1,165 @@
+| Export frequency | **[!UICONTROL Batch]**         | Batch destinations export files to downstream platforms in increments of three, six, eight, twelve, or twenty-four hours. Read more about [batch file-based destinations](/help/destinations/destination-types.md#file-based).                                                                                         |
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+>[!IMPORTANT]
+>
+>To connect to the destination, you need the **[!UICONTROL View Destinations]** and **[!UICONTROL Manage and Activate Dataset Destinations]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+In order to access your bucket on Experience Platform, you need to provide valid values for the following credentials:
+
+| Credential    | Description                                                                                              |
+|---------------|----------------------------------------------------------------------------------------------------------|
+| S3 Access key | The access key ID for your bucket. You can retrieve this value from the [!DNL Acxiom] team.              |
+| S3 Secret key | The secret key ID for your bucket. You can retrieve this value from the [!DNL Acxiom] team.              |
+| Bucket name   | This is your bucket where files will be shared. You can retrieve this value from the [!DNL Acxiom] team. |
+
+### New Account
+
+Define a new Acxiom Managed S3 location
+* **Connection type**
+  * **S3 access key (Required)** - Reference to Amazon S3 location, provided by admin when S3 role permissions are defined.  Must be equal to 20 characters and can only include characters A-Z and numbers 2-7.
+  * **S3 secret key (Required)** - Reference to Amazon S3 location, provided by admin when S3 role permissions are defined.  Must be equal to 40 characters and can only include characters A-Z, a-z, 0-9, +, \ and / .
+  * **Connect to destination** - Will use the provided authorization keys to connect to the S3 service to validate access.  Success will allow configuration of the destination details while failure will be indicated on the UI.
+
+![New Account](../../assets/catalog/advertising/acxiom/image-destination-new-account.png)
+
+### Existing Account
+
+Accounts already defined using the Acxiom Prospect-Suppression card will appear here for reuse.  These will appear in a list pop-up and when selected provides details on the account
```

Comment:
"Accounts already defined using the Acxiom Prospect-Suppression card will appear here for reuse" -> where is _here_? Please define that.

Reply:
dropped the the "here for reuse.  These will" we go on to describe the list so should be the same sentence.

Reply:
.. or we could call out that this view in the screenshot is from the UI if you navigate to **Destinations > Accounts**

Reply:
I can do that as well.

Reply:
Tried to reword this line to clarify further.